Dockerfile

```diff
@@ -34,7 +34,7 @@ COPY src src
 RUN ./gradlew -s bootWar
 
 # Run stage
-FROM openjdk:8-jre-alpine
+FROM adoptopenjdk/openjdk11:alpine-jre
```

Comment:
For our other images we use `openjdk:11-jre-slim`, could that be replicated here?

Reply:
Done.